facebook/OpenBIC
# Artemis: cb: Support ACCL cable power good timeout event

# Description
- Support ACCL cable power good timeout event.

# Motivation
- Support ACCL cable power good timeout event.

# Test Plan
- Build code: Pass
- Check ACCL cable power good timeout event: Pass

# Log
0    all      2018-03-09 04:53:58    pldmd            State Sensor: CB_SENSOR_ACCL_POWER_CABLE_1, no power good
0    all      2018-03-09 04:53:58    pldmd            State Sensor: CB_SENSOR_ACCL_POWER_CABLE_8, no power good
